packages/web/src/scss/components/Tooltip/_Tooltip.scss

```diff
@@ -1,21 +1,22 @@
-// 1. Although it seems pointless, having the arrow as a standalone HTML element has a purpose: the arrow can be
+// 1. Although it may seem pointless, having the arrow as a standalone HTML element has a purpose: the arrow can be
```

Comment:
```suggestion
// 1. Although it may seem pointless, having the arrow as a standalone HTML element. It has a purpose: the arrow can be
```

Make no sense as one sentence or comma is missing.